src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -738,7 +738,7 @@ def wait_for_operators_to_finish(self, timeout=consts.CLUSTER_INSTALLATION_TIMEO
             cluster_id=self.id,
             operators_count=len(operators_utils.filter_operators_by_type(operators, OperatorType.OLM)),
             operator_types=[OperatorType.OLM],
-            statuses=[consts.OperatorStatus.AVAILABLE, consts.OperatorStatus.FAILED],
+            statuses=statuses,
```

Comment:
why this change is needed? we have fall_on_error_status if you expect failure

Reply:
But we don't use those here. It always wait for failures regardless of `fall_on_error_status` value.
Also, If we don't mind if the operator is success or failure we can now use the `fall_on_error_status`

Reply:
why not using it? it does considered in _are_operators_in_status